template/page-toml-checker.html.jinja

```diff
@@ -1,32 +1,53 @@
 {% extends "base.html.jinja" %}
 
 {% block main %}
-<section class="container-fluid p-3">
-  <h1>xrp-ledger.toml Checker</h1>
+<section class="container-fluid">
+  <div class="p-3 pb-5">
+    <h1>xrp-ledger.toml Checker</h1>
 
-  <p>If you run an XRP Ledger validator or use the XRP Ledger for your business, you can provide information about your usage of the XRP Ledger to the world in a machine-readable <a href="https://developers.ripple.com/xrp-ledger-toml.html"><code>xrp-ledger.toml</code> file</a>.</p>
+    <p>If you run an XRP Ledger validator or use the XRP Ledger for your business, you can provide information about your usage of the XRP Ledger to the world in a machine-readable <a href="https://developers.ripple.com/xrp-ledger-toml.html"><code>xrp-ledger.toml</code> file</a>.</p>
 
-  <p>This tool allows you to verify that your <code>xrp-ledger.toml</code> file is syntactically
-    correct and deployed properly.</p><br/>
-  <form id="domain-entry">
-    <div class="input-group">
-      <input id="domain" type="text" class="form-control" required
-             placeholder="example.com (Domain name to check)"
-             pattern="^([a-zA-Z0-9]([a-zA-Z0-9\-]{0,61}[a-zA-Z0-9])?\.)+[a-zA-Z][a-zA-Z-]{0,22}[a-zA-Z]$"><br>
-      <button class="btn btn-primary form-control">Check toml file</button>
-    </div><!--/.input-group-->
-  </form>
-  <div id="result">
-    <h5 class='result-title'>Result</h5>
-      <ul id="log">
-    </ul>
+    <p>This tool allows you to verify that your <code>xrp-ledger.toml</code> file is syntactically
+      correct and deployed properly.</p><br/>
+    <form id="domain-entry">
+      <div class="input-group">
+        <input id="domain" type="text" class="form-control" required
+              placeholder="example.com (Domain name to check)"
+              pattern="^([a-zA-Z0-9]([a-zA-Z0-9\-]{0,61}[a-zA-Z0-9])?\.)+[a-zA-Z][a-zA-Z-]{0,22}[a-zA-Z]$"><br>
+        <button class="btn btn-primary form-control">Check toml file</button>
+      </div><!--/.input-group-->
+    </form>
+    <div id="result">
+      <h5 class='result-title'>Result</h5>
+        <ul id="log">
+      </ul>
+    </div>
   </div>
+  <div class="p-3 pt-5">
+      <h4>xrp-ledger domain verification</h4>
```

Comment:
This title and description isn't quite clear to difference from the domain checker. @mDuo13 any suggestions?

Reply:
```suggestion
      <h4>Look Up By Account</h4>
```